rasa_core/evaluate.py

```diff
@@ -8,16 +8,13 @@
 import argparse
 import io
 import logging
-import uuid
-from difflib import SequenceMatcher
 from tqdm import tqdm
-from typing import Text, List, Tuple
 
-from rasa_core import training
+from rasa_core import training, run
 from rasa_core import utils
 from rasa_core.agent import Agent
-from rasa_core.events import ActionExecuted, UserUttered
-from rasa_core.interpreter import RegexInterpreter, RasaNLUInterpreter
+from rasa_core.events import ActionExecuted
+from rasa_core.interpreter import NaturalLanguageInterpreter
 from rasa_core.trackers import DialogueStateTracker
 from rasa_core.training.generator import TrainingDataGenerator
 from rasa_nlu.evaluate import plot_confusion_matrix, log_evaluation_table
```

Comment:
we probably should not import anything from `rasa_nlu`?

Reply:
it's actually fine, rasa core depends on nlu (we also use things in other places from nlu) - the only things we shouldn't use are parts that depend on optional dependencies (e.g. spacy).